modules/local/summary_report.nf

```diff
@@ -0,0 +1,142 @@
+process SUMMARY_REPORT  {
+    label 'process_low'
+
+    conda "conda-forge::r-base=4.2.3 conda-forge::r-rmarkdown=2.22 conda-forge::r-tidyverse=2.0.0 conda-forge::r-knitr=1.43 conda-forge::r-dt=0.28 conda-forge::r-dtplyr=1.3.1 conda-forge::r-formattable=0.2.1 conda-forge::r-purrr=1.0.1 conda-forge::r-vegan=2.6_4 conda-forge::r-optparse=1.7.3 conda-forge::r-ggplot2=3.4.2 conda-forge::r-dplyr=1.1.2 conda-forge::r-data.table=1.14.8 conda-forge::r-patchwork=1.1.2"
```

Comment:
@tillenglert Did you let Daniel Lundin know that you produced such a container? I thought you two were collecting packages for this?